docs/development.rst

```diff
@@ -799,134 +799,80 @@ Adding a new species
 To add a new species to `stdpopsim` several things are required:
 
 1. The genome definition
-2. Default species parameters
-3. A genetic map with local recombination rates (optional)
+2. Generation time estimate
+3. Mutation rate (per generation)
+4. Recombination rate (per generation)
+5. Characteristic population size
 
-Once you have these things the first step is to create a new file in the `catalog`
-directory named for the species (see `Naming conventions`_ for more details). All
-code described below should go in this file unless explicitly specified otherwise.
-
---------------------------
-Default species parameters
---------------------------
-
-Four default parameters are required to create a new species:
-
-1. Generation time estimate
-2. Mutation rate (per generation)
-3. Recombination rate (per generation)
-4. Characteristic population size
+A genetic map with local recombination rates is optional.
 
 These parameters should be based on what values might be drawn from a typical population
 as represented in the literature for that species. Consequently one or more citations for
 each value are expected and will be required for constructing the species object detailed
 below.
 
+Once you have these things the first step is to create a new file in the `catalog/`
+directory named for the species (see `Naming conventions`_ for more details). All
+code described below should go in this file unless explicitly specified otherwise.
+
 -----------------------------------
 Adding/Updating a genome definition
 -----------------------------------
 
-A genome definition is created with a call to `stdpopsim.Genome()`  which requires a list
-of chromosomes and a citation for the assembly. `stdpopsim` has an automated procedure
-for obtaining this list from ensembl and saving it for automated parsing. First however
-the initial species directory must be created in the `stdpopsim/catalog` directory (e.g.
-`stdpopsim/catalog/AraTha`). Once that is done, run the `update_ensembl_data.py` script
-present in the top level directory providing the ensembl species id(s) as "_" delimited
-name(s) for positional arguments as shown below. If no positional arguments are specified
-then all specified registered in `stdpopsim` will be updated.
+`stdpopsim` has an automated procedure for generating a genome definition, which is
+accomplished by pulling data from Ensembl and saving it for parsing.
+First however the initial species directory must be created in the `stdpopsim/catalog/`
+directory (e.g. `stdpopsim/catalog/AraTha/`). Once that is done, we will hand the
```

Comment:
This has just been said above. But do we need to say that you should actually be in that directory now?